src/routes/safe/components/Apps/components/AppCard/index.tsx

```diff
@@ -15,6 +16,7 @@ const StyledAppCard = styled(Card)`
   height: 232px !important;
   box-sizing: border-box;
   cursor: pointer;
+  color: ${fontColor};
```

Comment:
one more question, why can't we use something linke so: `${({ theme }) => theme.colors.background}`?

Reply:
forgot it 😄 a good catch
I also added a declaration for SC theme